llvm/lib/Target/DirectX/DXIL.td

```diff
@@ -553,11 +553,10 @@ def Rbits :  DXILOp<30, unary> {
   let attributes = [Attributes<DXIL1_0, [ReadNone]>];
 }
 
-def CBits :  DXILOp<31, unary> {
+def CBits :  DXILOp<31, unaryBits> {
```

Comment:
Can we rename this to CountBits while we're at it? It's best if the OpCode names here match the existing [DXIL docs](https://github.com/microsoft/DirectXShaderCompiler/blob/main/docs/DXIL.rst#countbits) and the names used in DXC